GameServer/Logic/Content/ContentManager.cs

```diff
@@ -147,6 +149,29 @@ public string GetBuffScriptPath(string buffName)
             return GetContentPath(contentPackages, contentType, fileName);
         }
 
+        public string GetUnitStatPath(string model)
+        {
+            var contentType = "Stats";
+            var contentPackages = _content[contentType][model];
+            var fileName = string.Format("{0}/{1}.json", model, model);
+            return GetContentPath(contentPackages, contentType, fileName);
+        }
+
+        public string GetSpellDataPath(string spellName)
+        {
+            try
```

Comment:
Do we really need a try catch here? I don't think this is something that should silent fail. In fact this should fail with a custom exception type if no spell with the given name is found. `ContentNotFoundException` or similar.

Only realistic way I can imagine this failing is with a dictionary key not found, so I'd suggest just either checking for that with ifs, or only catching that exception type. Global exception catches are bad and cause silent fails when you don't want them.